### PR TITLE
CDAP-7548 Make IntegrationTest's jdbc connection work against secure clusters

### PR DESCRIPTION
**Pull request by anwar6953**
Update IntegrationTestManager to make getQueryClient's jdbc Connection work against ssl-enabled or authoriation-enabled cdap instances.

https://issues.cask.co/browse/CDAP-7548
http://builds.cask.co/browse/CDAP-RUT252-2

Tested against a CDAP instance with authorization enabled. The getQueryClient method fails without this fix.

### CODE REVIEW

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java [hunk 1]
```diff
@@ -417,8 +421,22 @@ public void deleteDatasetInstance(NamespaceId namespaceId, String datasetInstanc
   @Override
   public Connection getQueryClient(Id.Namespace namespace) throws Exception {
     ConnectionConfig connConfig = clientConfig.getConnectionConfig();
```
Comment (bdmogal) [inline cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java]: move this closer to usage?

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java [hunk 2]
```diff
@@ -417,8 +421,22 @@ public void deleteDatasetInstance(NamespaceId namespaceId, String datasetInstanc
...
+    AccessToken accessToken = clientConfig.getAccessToken();
+    if (accessToken != null) {
+      connParams.put(ExploreConnectionParams.Info.EXPLORE_AUTH_TOKEN.getName(), accessToken.getValue());
+    }
+    connParams.put(ExploreConnectionParams.Info.SSL_ENABLED.getName(),
+                   Boolean.toString(clientConfig.getConnectionConfig().isSSLEnabled()));
+    connParams.put(ExploreConnectionParams.Info.VERIFY_SSL_CERT.getName(),
+                   Boolean.toString(clientConfig.isVerifySSLCert()));
+
+    String connParamsString = Joiner.on("&").withKeyValueSeparator("=").join(connParams);
```
Comment (bdmogal) [inline cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java]: nit: better to not include datatype in variable name. can just use the `Joiner` in the `String.format` below, right?

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java [hunk 3]
```diff
@@ -417,8 +421,22 @@ public void deleteDatasetInstance(NamespaceId namespaceId, String datasetInstanc
...
+
+    String url = String.format("%s%s:%d?%s", Constants.Explore.Jdbc.URL_PREFIX, connConfig.getHostname(),
+                               connConfig.getPort(), connParamsString);
     return new ExploreDriver().connect(url, new Properties());
```
Comment (bdmogal) [inline cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java:438]: unrelated, but any reason why `ExploreDriver` is recreated for every call to `getQueryClient`? ignore if it is super lightweight.

**Review by bdmogal — APPROVED** [review]
minor nits. rest lgtm.

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java (hunk 3 above)
Comment (anwar6953) [reply cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java:438]: It seems to be a stateless class, so shouldn't be an issue to create a new instance each time.

**Review by bdmogal — APPROVED** [review]
lgtm

**Comment by anwar6953** [comment]
Squashed commits.

**Pull request merged**